static/tsconfig.json

```diff
@@ -5,7 +5,7 @@
         "name": "typescript-tslint-plugin",
       }
     ],
-    "lib": ["es2019"],
+    "lib": ["ES2021"],
```

Comment:
was this an intended update? why was it required

Reply:
oh, it's for replaceAll.

Reply:
yes, intended for the replaceAll!